benchmarks/osh-parser.sh

```diff
@@ -60,12 +60,19 @@ parser-task() {
   local times_out="$out_dir/$host.$job_id.times.csv"
 
   local shell_name
-  shell_name=$(basename $sh_path)
+  case $sh_path in
+    _bin/*/mycpp-souffle/*)
+      shell_name=osh-native-souffle
```

Comment:
Hm the relationship between sh_path and shell_name is a smell I remember juggling ... 

I think there is a place in R where we have a heuristic for `osh-native` as well

hm